docs/docs-manifest.json

```diff
@@ -1068,11 +1068,6 @@
       ],
       "categories": []
     },
-    {
-      "category_slug": "utilities",
-      "category_title": "Utilities",
-      "categories": []
-    },
```

Comment:
Any specific reason you deleted this?

Reply:
Hi @louwie17,

A linter rule forced me to regenerate the `manifest.json` (there is documentation [here](https://github.com/woocommerce/woocommerce/blob/trunk/docs/contributing-docs/contributing-docs.md#regenerating-the-docs-manifest)).
I think that happened because I modified [this file](https://github.com/woocommerce/woocommerce/pull/49723/files#diff-a521d8cb1c35dd723ee31caad7d981742190a00dad7d7b4c81c6d69fc9032664R42).

The `Utilities` category was deleted as part of the manifest regeneration.

Reply:
Gotcha, thanks for the clarification, this is fine than, feel free to merge!